src/main/java/net/minecraftforge/network/ServerStatusPing.java

```diff
@@ -58,21 +66,50 @@
  *          }
  *     ]
  * }
+ * </pre>
+ * </code>
```

Comment:
This should use a `{@code}` block tag wrapped in `<pre>`: (this is the closing for the preceding review comment)
```suggestion
 * }</pre>
```